data/language/en-GB.txt

```diff
@@ -3682,7 +3682,7 @@ STR_5340    :{SMALLFONT}{BLACK}Clearance height
 STR_5341    :Flags
 STR_5342    :Choose a map tile
 STR_5343    :Automatically place staff
-STR_5344    :Changelog
+STR_5344    :<not used anymore>
```

Comment:
Why are you removing this one and adding a new entry below. All your doing is throwing out work that the translators do.